DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/SectionContentViewModelTests.cs

```diff
@@ -618,5 +618,20 @@ bool separatorExpected
             // Then
             sectionContentViewModel.DisplayPostLearningSeparator.Should().Be(separatorExpected);
         }
+
+        [Test]
+        public void Section_content_should_properly_convert_consolidation_path()
```

Comment:
How is this test different to the other test? 🙂

Reply:
The other test is doing some sort of check with "courseSettings" to make sure it has a consolidation path. I added this test because it would test that the consolidation path is being properly converted so if the courseSettings thing fails, we know its due to something else and not the conversion. We could change the courseSettings test to just check if consolidation path is not null? So that these tests are both separate. Does that make sense?

Reply:
The course settings is irrelevant to the consolidation path. So we only need one of these tests 🙂

Reply:
Ah okay yep looks like a the coursesettings test was changed/added by mistake, will fix that now